terraform/modules/kubernetes/azure_redis.tf

```diff
@@ -70,18 +82,20 @@ resource "azurerm_redis_cache" "redis-queue" {
   sku_name                      = var.redis_queue_sku_name
   minimum_tls_version           = var.redis_minimum_tls_version
   public_network_access_enabled = var.redis_public_network_access_enabled
+  zones                         = var.redis_replica_zones
 
   redis_configuration {
-    maxmemory_policy = "noeviction"
+    maxmemory_policy                = "noeviction"
+    aof_backup_enabled              = var.redis_backup_enable
```

Comment:
As discussed, we will not configure data persistence right now, only multi AZ support
Let's keep a copy of the branch as it might be useful later on

Reply:
I thought we were configuring both?